Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -28,6 +28,18 @@ public enum MouseButton
     public MouseButton MouseLookButton = MouseButton.Shift;
     public bool IsControllerLookInverted = true;
 
+    public enum ControlMode
```

Comment:
@NeerajW yes indeed this class (ManualCameraControl) is not in the HolotoolKit namespace
Happy to "fix that" but it will probably make more sense to do this change in another PR
@jwittner sounds good then even if we keep it inside the class like that we'll have to keep "CurrentControlMode" which I guess is not a big deal anyway.